Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -958,6 +1127,58 @@ def route_edl() -> Response:
     return resp
 
 
+@APP.route('/log', methods=['GET'])
+def route_edl_log() -> Response:
+    """Show the edl indicators log on '/log' API request. """
+    params = demisto.params()
+
+    cache_refresh_rate: str = params.get('cache_refresh_rate')
+    auth_resp = authenticate_app(params, request.headers)
+    if auth_resp:
+        return auth_resp
+
+    created = datetime.now(timezone.utc)
+    request_args = get_request_args(request.args, params)
+    demisto.debug("Getting log file to show")
+    edl_data_log = ''
+    if os.path.exists(EDL_FULL_LOG_PATH):
+        demisto.debug("found log file")
+        with open(EDL_FULL_LOG_PATH, 'r') as log_file:
+            log_file.seek(0)
+            edl_data_log = log_file.read()
```

Comment:
Correct me if I am wrong but when we work with the edl we try never to hold the entire list in memory of indicators in memory to prevent memory issues. Seems that we are doing it here which can cause memory issues.

Reply:
We do hold the entire list in the memory.
In the function route_edl you can see that the response edl_data is sent as a string.

I understand your concern, but the log data should be comparable in size with the indicator list itself. From what I gathered in slack, it seems customers did not face memory issues with the indicator list, so I think this is an okay start.